pkg/awsutils/awsutils.go

```diff
@@ -700,7 +700,11 @@ func (cache *EC2InstanceMetadataCache) createENI(useCustomCfg bool, sg []*string
 
 	if useCustomCfg {
 		log.Info("Using a custom network config for the new ENI")
-		input.Groups = sg
+		if len(sg) != 0 {
+			input.Groups = sg
+		} else {
+			log.Warnf("No custom networking security group found, will use same as the node: %s", input.Groups)
```

Comment:
@jayanthvn, could you please change this wording to: "No custom networking security group found, will use the node's primary ENI's SGs:"